pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -331,9 +331,10 @@ func (a *Bootstrap) addSystemdUnits(uri string, templateData *bootstrapTemplateD
 		"systemd-journal-gatewayd.socket": {},
 		"approve-csr.service":             {},
 		// baremetal & openstack platform services
-		"keepalived.service": {},
-		"coredns.service":    {},
-		"ironic.service":     {},
+		"keepalived.service":        {},
+		"coredns.service":           {},
+		"ironic.service":            {},
+		"master-bmh-update.service": {},
```

Comment:
Can there be some wants/depends_on relationship between these 2 services that allows us to not activate each individual service? or are these actually 2 independent services?

Reply:
There are 2 independent .service files for them if that's what you mean.  I did try it with a dependency but the service was just marked as disabled and wouldn't start.

Reply:
master-bmh-update talks to ironic, but they are independent with different goals. I think it's more obvious to leave both explicitly enabled